README.rst

```diff
@@ -2,22 +2,56 @@
  Crate JMX HTTP Exporter
 =========================
 
-This project implements a `javaagent` which exposes CrateDB specific metrics over
-HTTP.
-
-.. warning::
-
-   THIS PROJECT IS CURRENTLY IN EARLY ALPHA STATE!
+This project implements a `javaagent` which exposes CrateDB specific JMX MBean
+metrics over HTTP.
 
 Usage
 =====
 
+`Download the jar`_ and enable the agent using the `CRATE_JAVA_OPTS` environment
+variable.
+
+Example usage:
+
 ::
 
-   export CRATE_JAVA_OPTS="-javaagent:<PATH_TO>/crate_jmx_exporter-0.1.0.jar=<HOST>:<PORT>"
+   export CRATE_JAVA_OPTS="-javaagent:<PATH_TO>/crate-jmx-exporter-0.1.0.jar=8080"
    ./bin/crate
 
-Afterwards the metrics can be fetched via HTTP under the configured host and port.
+Afterwards the `Available HTTP endpoints`_ are accessible via HTTP at
+http://localhost:8080/<ENDPOINT>.
+
+To bind the java agent to a specific IP change the port number to ``host:port``.
+
+Available HTTP endpoints
+------------------------
+
+/metrics
+~~~~~~~~
+
+This endpoint will respond general JVM and CrateDB specific JMX MBean metrics as
```

Comment:
"this endpoint will return general" 

"respond general" is a bit odd so maybe "return general" is a bit better